### DISCUSSION TITLE
Questions about source maps...

### DISCUSSION THREAD

**Discussion by Elindo586** [vercel/next.js#40163] [Help] [answered]
So I am getting this from lighthouse:

```
Missing source maps for large first-party JavaScript

Source maps translate minified code to the original source code. This helps developers debug in production. In addition, Lighthouse is able to provide further insights. Consider deploying source maps to take advantage of these benefits. [Learn more](https://developers.google.com/web/tools/chrome-devtools/javascript/source-maps?utm_source=lighthouse&utm_medium=devtools).

<html>
<body>
<!--StartFragment-->

chrome-extension://fmkadmapgofadopljbjfkapdkoienihi/build/react_devtools_backend.js |  
-- | --
Large JavaScript file is missing a source map

<!--EndFragment-->
</body>
</html>
```
^^^ 
Any ideas of what this all means?  

If we are talking about javascript, then I can think I am using Nextjs, and the other library is from GSAP to do some animations...

Does Nextjs generates a source map for the code?   What recommendations would be good to take care of this?

**Comment by icyJoseph** [comment, marked as answer]
Hi,

Could you please run your lighthouse analysis in Incognito mode and report back? I think it has ot do with chrome extensions, but I am not 100% sure.

**Reply by Elindo586** [reply]
Ok... it doesn't show the error in incognito. 

There are other more simple errors that I have to fix, but I don't see the source map error....

Why do you think it doesn't show in the incognito window?

**Reply by icyJoseph** [reply]
It is because the Chrome extension called React Dev Tools is messing up with your measurements. It is always the case that lighthouse runs are best done in incognito mode :) - And the best way is through the lighthouse CLI with a proper CPU slowdown factor.
